.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: osx
   pool:
-    vmImage: macOS-13
+    vmImage: macOS-15
```

Comment:
That's the change I get in depending downstream repos already.

I think HDF5 as published right now injects a hard-coded dependency on `'/Applications/Xcode_15.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/usr/lib/libpthread.tbd', needed by ... [dependent targets]`

Reply:
I don't think this is in HDF5. Maybe in a dependency of hdf5. Can you check which file in the environment has that path hard-coded?

Reply:
Running
```
CONDA_OVERRIDE_OSX=11.0 CONDA_SUBDIR=osx-64 conda create -n a warpx
conda activate a
grep -Iir "Xcode_15.2.app" $CONDA_PREFIX
```
on my Linux machine shows that it is OpenPMD's fault.

Reply:
Thank you so much! Then I try to get https://github.com/conda-forge/openpmd-api-feedstock/pull/127 to get green :pray: